account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py

```diff
@@ -56,13 +57,10 @@ def _create_move_and_lines(
             self, amount, debit_account_id, credit_account_id,
             sums, label, form, partner_id, currency_id,
             analytic_debit_acc_id=False, analytic_credit_acc_id=False):
-
-        reversable = form.journal_id.company_id.reversable_revaluations
         base_move = {
             'name': label,
             'journal_id': form.journal_id.id,
             'date': form.revaluation_date,
-            'to_be_reversed': reversable
         }
```

Comment:
@goliveirab 
```
if form.journal_id.company_id.reversable_revaluations:
   base_move['auto_reverse'] = True
   base_move['reverse_date'] = form.revaluation_date + timedelta(days=1)
```

> Why did you remove the dependency on account-reversal ?
> The reason we did this is that you must reverse all revaluation entries next day like you do for any cut-off entries.

@fclementic2c  Let us use core reversal feature. It will take in charge of reversing the regarding journal entries if  the settings in the company were set. They will be scheduled to be reversed on next date.

https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_move.py#L442-L453
https://github.com/odoo/odoo/blob/12.0/addons/account/data/service_cron_reverse.xml

> Of course you can do it manually but... why would you?
Some localizations require not to reverse the Currency Realization.

Reply:
> @fclementic2c Let us use core reversal feature. It will take in charge of reversing the regarding journal entries if the settings in the company were set. They will be scheduled to be reversed on next date.
> 
> https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_move.py#L442-L453
> https://github.com/odoo/odoo/blob/12.0/addons/account/data/service_cron_reverse.xml

I agree. Reversal is now a standard feature and can replace the community module.

Reply:
>Some localizations require not to reverse the Currency Realization.

If you do not reverse entries, reval. entries generated will be just wrong.
As you said it yourself the cumulative currency balance is not amended with the revaluation entries. So if you try to run the wizard the following month (for instance) on the same account without reversing previous reval. entries then new reval. computation will be wrong.